src/bp/jest-before.ts

```diff
@@ -15,6 +15,10 @@ if (!process.BOTPRESS_EVENTS) {
   process.BOTPRESS_EVENTS = new EventEmitter()
 }
 
+if (!process.APP_DATA_PATH) {
```

Comment:
**pointing the attention of reviwers on this line**

Reply:
No issue there, I suggested this change instead of having `process.APP_DATA_PATH || ''`everywhere in the runtime code, just because the app data path is not set when running unit tests